CHANGELOG.md

```diff
@@ -31,14 +31,19 @@
   [**@Sherwin-14**](https://github.com/betolink),
   [**@mfisher87**](https://github.com/mfisher87)
 
-
-
 ### Fixed
 
 - Removed Broken Link "Introduction to NASA earthaccess"
   ([#779](https://github.com/nsidc/earthaccess/issues/779))
   ([**@Sherwin-14**](https://github.com/Sherwin-14))
 
+### Removed
+
+- `binder/` directory as we no longer need a special [binder](https://mybinder.org)
+  environment with the top-level `environment.yml` introduce in
+  [#733](https://github.com/nsidc/earthaccess/issues/733)
+  ([@jhkennedy](https://github.com/jhkennedy))
```

Comment:
I don't feel too strongly about this, but I'd be OK omitting this from the change log. It's not really user facing. For developers, we have PRs and commits as history, so I don't feel we need things like this in changelog.